pkg/operator/ceph/file/dependent_test.go

```diff
@@ -75,8 +75,27 @@ func TestCephFilesystemDependents(t *testing.T) {
 	) (client.SubvolumeList, error) {
 		return client.SubvolumeList{}, nil
 	}
+	noCephFilesystem := func(
+		context *clusterd.Context, clusterInfo *client.ClusterInfo, fsName string,
+	) (*client.CephFilesystemDetails, error) {
+		return nil, errors.Errorf("not found")
```

Comment:
@travisn if we use error code to replace the string, how to mock the error repsones?

Reply:
yes